src/modules/communication/internals/twilio-params.ts

```diff
@@ -0,0 +1,49 @@
+import { PhoneNumberUtil } from 'google-libphonenumber';
+import {
+  PhoneNumber,
+  SendSMSParams,
+  ValidationError,
+  ValidationFailure,
+} from '../types';
+
+export default class SMSParams {
+  public static phoneNumberToString(phoneNumber: PhoneNumber): string {
+    return `${phoneNumber.countryCode}${phoneNumber.phoneNumber}`;
+  }
+
+  public static validate(params: SendSMSParams): void {
+    const failures: ValidationFailure[] = [];
+
+    const phoneUtil = <{
+      isValidNumber: (a: string) => boolean;
+      parse: (a: string) => string
+    }>(<{
+      getInstance:() => any
+    }>PhoneNumberUtil).getInstance();
```

Comment:
Can we make it more readable?

Reply:
agreed @akashlakhera  sir, will make required changes.